skyvern/webeye/browser_factory.py

```diff
@@ -475,10 +475,10 @@ async def _create_cdp_connection_browser(
             raise Exception("Port 9222 is already in use. Another process may be using this port.")
 
         browser_process = subprocess.Popen(
-            [browser_path, "--remote-debugging-port=9222"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+            [browser_path,"--remote-debugging-port=9222", "--no-first-run", "--no-default-browser-check"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
```

Comment:
```suggestion
            [browser_path, "--remote-debugging-port=9222", "--no-first-run", "--no-default-browser-check"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
```

this should have been caught by our linter. if you haven't set up pre-commit, see [here](https://github.com/Skyvern-AI/skyvern/?tab=readme-ov-file#additional-setup-for-contributors)

Reply:
resolved, it as per the pre-commit requirment